cmd/manager/main.go

```diff
@@ -91,10 +94,15 @@ func main() {
 
 	ctx := context.TODO()
 
+	// We sometimes startup faster than we can reach kube-api. Poll on failure to prevent us terminating
 	// Become the leader before proceeding
-	err = leader.Become(ctx, "knative-serving-operator-lock")
-	if err != nil {
-		log.Error(err, "")
+	if perr := wait.PollImmediate(time.Second, 60*time.Second, func() (bool, error) {
```

Comment:
@yu2003w I do not think we will need the leader selection any longer, coz the new controller model does not need it.

Reply:
ok, I will close this. Thanks.